src/Seq.Input.RabbitMQ/RabbitMQInput.cs

```diff
@@ -80,6 +80,12 @@ public class RabbitMQInput : SeqApp, IPublishJson, IDisposable
             HelpText = "Whether or not messages should be auto-acknowledged. The default is true.")]
         public bool IsReceiveAutoAck { get; set; } = true;
 
+        [SeqAppSetting(
+            DisplayName = "Dead Letter Exchange",
+            IsOptional = true,
+            HelpText = "Name of dead letter exchange associated with this queue.")]
+        public string Dlx { get; set; } = "guest";
```

Comment:
I think this shouldn't have a default value, and should be ignored if it's null or empty.

Reply:
only just saw this, totally agree, l have no idea how that ended up there, makes no sense